cookbook/import_export/create-connector.rst

```diff
@@ -43,40 +31,80 @@ Register the bundle in AppKernel:
+        jobs:
+            demo_job:
+                title: acme_connector.jobs.demo_job.title
+                type:  export
+                steps:
+                    stepOne:
+                        title:         acme_connector.jobs.demo_job.export.title
+                        services:
+                            reader:    pim_connector.reader.dummy_item
+                            processor: pim_connector.processor.dummy_item
+                            writer:    pim_connector.writer.dummy_item
 
-Configure a job in ``Resources/config/batch_jobs.yml``:
+Here we use some existing dummy reader, processor and writer (they implement relevant interfaces and are useable but they do nothing with data).
 
-.. literalinclude:: ../../src/Acme/Bundle/DemoConnectorBundle/Resources/config/batch_jobs.yml
-   :language: yaml
-   :linenos:
-   :lines: 1-13
+The reader is implemented in the class ``Pim\Component\Connector\Reader\DummyItemReader``, it's defined as a service in the ConnectorBundle with the alias ``pim_connector.reader.dummy_item`` in the file ``Resources\config\readers.yml``.
 
-Here we use some existing readers, processors and writers from native csv product export, they are defined as services in config files of the PimBaseConnectorBundle, we'll see later how to create your own elements.
+The processor is implemented in the class ``Pim\Component\Connector\Processor\DummyItemProcessor``, it's defined as a service in the ConnectorBundle with the alias ``pim_connector.processor.dummy_item`` in the file ``Resources\config\processors.yml``.
 
-Title keys can be translated in ``messages.en.yml``
+The writer is implemented in the class ``Pim\Component\Connector\Writer\DummyItemWriter``, it's defined as a service in the ConnectorBundle with the alias ``pim_connector.writer.dummy_item`` in the file ``Resources\config\writers.yml``.
 
-.. literalinclude:: ../../src/Acme/Bundle/DemoConnectorBundle/Resources/translations/messages.en.yml
-   :language: yaml
-   :linenos:
-   :lines: 1-6
+We'll explain in next cookbook chapters how to create your own elements with real logic inside.
 
-Use our new Connector
+Translate Job and Step titles
+-----------------------------
+
+Create a file ``Resources/config/messages.en.yml`` in our Bundle to translate title keys.
+
+.. code-block:: yaml
+
+    acme_connector:
+        jobs:
+            demo_job:
+                title: Demo Job
+                stepOne:
+                    title: First Step
+
+Create a Job Instance
 ---------------------
 
-Now if you refresh cache, your new export can be found under Spread > Export profiles > Create export profile.
+Each Job can be configured through a JobInstance, an instance of the Job.
+
+Means that we can define a job and many instances of this one, with different configurations.
 
-You can run the job from UI or you can use following command:
+Our demo job does not take any configuration, we can create an instance with the following command:
```

Comment:
Maybe it would be clearer to write it in another sentence : "Please note that this job instance does not take any configuration"
After or before the code block.